docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -145,232 +320,273 @@ where:
+    It would be great to implement others and test the impact on precipitation.
+    See for example [Wood\_2005](https://journals.ametsoc.org/doi/full/10.1175/JAS3530.1)
+    Table 1 for other simple choices.
+
+### Snow autoconversion
 
-## Accretion
+Snow autoconversion defines the rate of conversion form cloud ice to snow due
+the growth of cloud ice by sublimation (deposition of water vapor).
+It is defined as the change of mass of cloud ice for cloud ice particles
+larger than threshold ``r_{is}``.
+See [Harrington_1995](https://journals.ametsoc.org/doi/pdf/10.1175/1520-0469%281995%29052%3C4344%3APOICCP%3E2.0.CO%3B2)
+for derivation.
 
-Accretion defines the rate of conversion from cloud to rain water resulting from collisions between cloud droplets and rain drops. It is parameterized following Kessler 1995:
 ```math
 \begin{equation}
-\left. \frac{d \, q_{rai}}{dt} \right|_{accr} = \int_0^\infty n(r) \, \pi r^2 \, v_{drop} E_{col} q_{liq} dr
+  \left. \frac{d \, q_{sno}}{dt} \right|_{acnv} =
+  \frac{1}{\rho} \frac{d}{dt} \left( \int_{r_{is}}^{\infty} m(r) n(r) dr \right) =
+  \left. \frac{1}{\rho} \frac{dr}{dt} \right|_{r=r_{is}} m(r_{is}) n(r_{is}) + \frac{1}{\rho} \int_{r_{is}}^{\infty} \frac{dm}{dt} n(r) dr
 \end{equation}
 ```
-where:
- - ``E_{col}`` is the collision efficiency,
- - ``v_{drop}`` is defined in eq.(\ref{eq:vdrop}).
+The ``\frac{dm}{dt}`` is obtained by solving the water vapor diffusion equation in spherical coordinates
+and linking the changes in temperature at the drop surface to the changes in saturated vapor pressure via the Clausius-
+Clapeyron equation, following
+[Mason\_1971](https://global.oup.com/academic/product/the-physics-of-clouds-9780199588046?cc=us&lang=en&).
 
-Integrating over the distribution and using the RWC to eliminate the ``\lambda_{MP}`` results in:
+For the simplest case of spherical particles and not taking into account ventilation effects:
 ```math
 \begin{equation}
-\left. \frac{d \, q_{rai}}{dt} \right|_{accr}  = \Gamma \left(\frac{7}{2} \right) \pi^{1/8} 8^{-7/8} E_{col} v_c(\rho) \, \left(\frac{\rho}{\rho_{water}}\right)^{7/8} n_{0_{MP}}^{1/8} g^{1/2} q_{liq} q_{rai}^{7/8} = A(\rho) \, n_{0_{MP}}^{1/8} g^{1/2} q_{liq} q_{rai}^{7/8}
+\frac{dm}{dt} = 4 \pi \, r \, (S - 1) \, G(T, p)
+\label{eq:mass_rate}
 \end{equation}
 ```
-The default value of collision efficiency ``E_{coll}`` is set to 0.8 so that the resulting accretion rate is close to the empirical accretion rate in Smolarkiewicz and Grabowski 1996. Assuming a constant ``E_{col}`` is an approximation, see for example [collision efficiency](https://journals.ametsoc.org/doi/10.1175/1520-0469%282001%29058%3C0742%3ACEODIA%3E2.0.CO%3B2).
+where:
+ - ``S(q_{vap}, q_{vap}^{sat}) = \frac{q_{vap}}{q_{vap}^{sat}} - 1`` is commonly labeled as supersaturation,
+ - ``q_{vap}^{sat}`` is the saturation vapor specific humidity,
+ - ``G(T) = \left(\frac{L_s}{KT} \left(\frac{L_s}{R_v T} - 1 \right) + \frac{R_v T}{p_{vap}^{sat} D} \right)^{-1}``
```

Comment:
Is `G` a function of temperature and pressure or just temperature? Notation is inconsistent. The equations mostly have `G(T,p)` but there is no pressure here in the definition.

Reply:
corrected - thanks!